athf/commands/metrics.py

```
@@ -0,0 +1,326 @@
+        "hunt_id": "--hunt",
+        "session_id": "--session",
+        "custom": "(custom fields are derived automatically from --field)",
+    }
+    known_kwargs: Dict[str, Any] = {}
+    custom: Dict[str, Any] = {}
+    for k, v in parsed.items():
+        if k in reserved:
+            raise click.BadParameter(
+                f"--field {k!r} collides with a dedicated option; "
+                f"use {reserved[k]} instead"
+            )
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)